examples/vector_databases/Using_vector_databases_for_embeddings_search.ipynb

```diff
@@ -1152,6 +1175,469 @@
+    "\n",
+    "[redis-url]: https://redis.com\n",
+    "\n",
+    "[redis-py-url]: https://github.com/redis/redis-py\n",
+    "[redis-py-stars]: https://img.shields.io/github/stars/redis/redis-py.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "[redis-py-package]: https://pypi.python.org/pypi/redis\n",
+    "\n",
+    "[jedis-url]: https://github.com/redis/jedis\n",
+    "[jedis-stars]: https://img.shields.io/github/stars/redis/jedis.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "[Jedis-package]: https://search.maven.org/artifact/redis.clients/jedis\n",
+    "\n",
+    "[nredisstack-url]: https://github.com/redis/nredisstack\n",
+    "[nredisstack-stars]: https://img.shields.io/github/stars/redis/nredisstack.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "[nredisstack-package]: https://www.nuget.org/packages/nredisstack/\n",
+    "\n",
+    "[node-redis-url]: https://github.com/redis/node-redis\n",
+    "[node-redis-stars]: https://img.shields.io/github/stars/redis/node-redis.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "[node-redis-package]: https://www.npmjs.com/package/redis\n",
+    "\n",
+    "[redis-om-python-url]: https://github.com/redis/redis-om-python\n",
+    "[redis-om-python-author]: https://redis.com\n",
+    "[redis-om-python-stars]: https://img.shields.io/github/stars/redis/redis-om-python.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "\n",
+    "[redisearch-go-url]: https://github.com/RediSearch/redisearch-go\n",
+    "[redisearch-go-author]: https://redis.com\n",
+    "[redisearch-go-stars]: https://img.shields.io/github/stars/RediSearch/redisearch-go.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "\n",
+    "[redisearch-api-rs-url]: https://github.com/RediSearch/redisearch-api-rs\n",
+    "[redisearch-api-rs-author]: https://redis.com\n",
+    "[redisearch-api-rs-stars]: https://img.shields.io/github/stars/RediSearch/redisearch-api-rs.svg?style=social&amp;label=Star&amp;maxAge=2592000\n",
+    "\n",
+    "\n",
+    "In the below cells, we will walk you through using Redis as a vector database. Since many of you are likely already used to the Redis API, this should be familiar to most."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "698e24f6",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "There are many ways to deploy Redis with RediSearch. The easiest way to get started is to use Docker, but there are are many potential options for deployment. For other deployment options, see the [redis directory](./redis) in this repo.\n",
+    "\n",
+    "For this tutorial, we will use Redis Stack on Docker.\n",
+    "\n",
+    "Start a version of Redis with RediSearch (Redis Stack) by running the following docker command\n",
+    "\n",
+    "```bash\n",
+    "$ docker run -d -p 6379:6379 -p 8001:8001 redis/redis-stack:latest\n",
```

Comment:
I think it would be better for us to link to a `docker-compose.yml` file nestled under the `/redis` subdirectory. It's a pattern Weaviate uses. I like that because it creates an artifact that teams can then take with them and use in their stack (the compose file) -- and we can instill some best practices there.